includes/settings.php

```diff
@@ -1,103 +1,264 @@
-		<form method="post" action="options.php">
-		
-			<?php settings_fields('stripe_settings_group'); ?>
-			
-			<table class="form-table">
-				<tbody>
-					<tr valign="top">	
-						<th scope="row" valign="top">
-							<?php _e('Test Mode', 'simple-stripe-payments'); ?>
-						</th>
-						<td>
-							<input id="stripe_settings[test_mode]" name="stripe_settings[test_mode]" type="checkbox" value="1" <?php checked(1, $stripe_options['test_mode']); ?> />
-							<label class="description" for="stripe_settings[test_mode]"><?php _e('Check this to use the plugin in test mode.', 'test_mode'); ?></label>
-						</td>
-					</tr>
-				</tbody>
-			</table>	
-			
-			<h3 class="title"><?php _e('API Keys', 'simple-stripe-payments'); ?></h3>
-			<table class="form-table">
-				<tbody>
-					<tr valign="top">	
-						<th scope="row" valign="top">
-							<?php _e('Live Secret', 'simple-stripe-payments'); ?>
-						</th>
-						<td>
-							<input id="stripe_settings[live_secret_key]" name="stripe_settings[live_secret_key]" type="text" class="regular-text" value="<?php echo $stripe_options['live_secret_key']; ?>"/>
-							<label class="description" for="stripe_settings[live_secret_key]"><?php _e('Paste your live secret key.', 'simple-stripe-payments'); ?></label>
-						</td>
-					</tr>
-					<tr valign="top">	
-						<th scope="row" valign="top">
-							<?php _e('Live Publishable', 'simple-stripe-payments'); ?>
-						</th>
-						<td>
-							<input id="stripe_settings[live_publishable_key]" name="stripe_settings[live_publishable_key]" type="text" class="regular-text" value="<?php echo $stripe_options['live_publishable_key']; ?>"/>
-							<label class="description" for="stripe_settings[live_publishable_key]"><?php _e('Paste your live publishable key.', 'simple-stripe-payments'); ?></label>
-						</td>
-					</tr>
-					<tr valign="top">	
-						<th scope="row" valign="top">
-							<?php _e('Test Secret', 'simple-stripe-payments'); ?>
-						</th>
-						<td>
-							<input id="stripe_settings[test_secret_key]" name="stripe_settings[test_secret_key]" type="text" class="regular-text" value="<?php echo $stripe_options['test_secret_key']; ?>"/>
-							<label class="description" for="stripe_settings[test_secret_key]"><?php _e('Paste your test secret key.', 'simple-stripe-payments'); ?></label>
-						</td>
-					</tr>
-					<tr valign="top">	
-						<th scope="row" valign="top">
-							<?php _e('Test Publishable', 'simple-stripe-payments'); ?>
-						</th>
-						<td>
-							<input id="stripe_settings[test_publishable_key]" name="stripe_settings[test_publishable_key]" class="regular-text" type="text" value="<?php echo $stripe_options['test_publishable_key']; ?>"/>
-							<label class="description" for="stripe_settings[test_publishable_key]"><?php _e('Paste your test publishable key.', 'simple-stripe-payments'); ?></label>
-						</td>
-					</tr>
-				</tbody>
-			</table>	
+class Settings {
```

Comment:
Class `Settings` is generic name, add a prefix to the name so that it does not conflict with any other class.